subprojects/core-api/src/main/java/org/gradle/api/tasks/TaskContainer.java

```diff
@@ -161,6 +165,7 @@
      * @param constructorArgs The arguments to pass to the task constructor
      * @return The newly created task object
      * @throws InvalidUserDataException If a task with the given name already exists in this project.
+     * @throws NullPointerException If any of the values in {@code constructorArgs} is null.
```

Comment:
This should also be mentioned in the first paragraph of the Javadoc.

Reply:
done